README.md

```diff
@@ -22,14 +22,40 @@ Recommendations: Docker-Machine (on [DigitalOcean with 2GB memory][do-tut])
 
    [do-tut]: https://www.digitalocean.com/community/tutorials/how-to-provision-and-manage-remote-docker-hosts-with-docker-machine-on-ubuntu-16-04
 
+
+### Running with docker-compose:
+
+Before running docker-compose up for the first time,
+either do a pull (faster):
+
+`docker-compose pull`
+
+or do a build (to utilize recent or local changes):
+
+`GIT_HASH $(git rev-parse --short HEAD) docker-compose up --detach`
+
+subsequently you should only need to run:
+
+`docker-compose up --detach`
+
+To force a full re-build with no cache from previous builds:
+`docker-compose build --build-arg GIT_HASH $(git rev-parse --short HEAD) --parallel --no-cache`
+
+And to stop:
+`docker-compose down`
+
+_(or Ctrl+C if you did not run with --detach)_
+
+### check your ip (only necessary on docker-machine):
 ```
-GIT_HASH=$(git log --pretty="%h" -n 1) docker-compose up --build --detach
 docker-machine ip
 >>> 123.45.67.89
 ```
 
 Visit your instance at: `http://123.45.67.89.xip.io/`
 
+Or visit a native docker instance at `http://localhost:80/`
```

Comment:
Was trying to figure right terms for talking about docker: is "native" docker (local) and docker-machine (remote) the best differentiation to stick with in our docs?

Reply:
I just made up the "native docker" phrase. I hope that makes sense. I mean that if you are running docker on a linux machine, it is running "natively" and not via docker-machine (no VM). Such that ports exposed are directly available on localhost. This is typically the case on a remote (linux) server as well. So it's not analogous to "local" vs "remote". I could include some clarifying text here.

Reply:
ok! i can clarify it later when i do the docker-machine docs, if that works for you. I'd count this as resolved